dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+# Created by Barbara Ellynes Zucchi Nobre Silva for the newsmap package: https://github.com/koheiw/newsmap
+
+AFRICA:
+  EAST:
+    'BI': [Burundi, Burúndi, Burundian*, Burundiense*, Burundinês*, Bujumbura]
+    'DJ': [Djibouti, Jibuti, Djiboutian*]
+    'ER': [Eritreia, Eritreu*, Eritréia*, Eritreense*, Asmara]
+    'ET': [Etiópia, Etíope*, Etiopian*, Adis Abeba]
+    'KE': [Quénia, Quênia, Quenian*, Nairóbi]
+    'KM': [Comores, Comoriense*, Comorian*, Moroni]
+    'MG': [Madagáscar, Madagascar, Madagascarense*, Malgaxe*, Antananarivo]
+    'MU': [Maurícia, Maurício, Maurician*, Porto Luís]
+    'MW': [Malawi, Maláui, Malaui, Malauian*, Malawian*, Malavian*, Malawiense*, Malauiense*, Malavita*, Lilongué, Lilôngue, Lilóngue, Lilongue, Lilongwe]
+    'MZ': [Moçambique, Moçambican*, Maputo]
+    'RE': [Reunião, Ilha da Reunião, Reunionense*, Reunionês*, Reuniones*]
+    'RW': [Ruanda, Ruandês*, Quigali]
+    'SC': [Seicheles, Seychelles, Seychellense*, Seichelense*]
+    'SO': [Somália, Somali*, Mogadíscio, Mogadixo]
+    'TZ': [Tanzânia, Tanzania, Tanzanian*, Dodoma, Dar es Salaam, Dar es Salã]
+    'UG': [Uganda, Ugandense*, Ugandês*, Campala]
+    'YT': [Mayotte, Maiote, Maiotense*, Mamudzu]
+    'ZM': [Zâmbia, Zambian*, Lusaca]
+    'ZW': [Zimbabwe, Zimbábue, Zimbabué, Zimbaué, Zimbabuean*, Zimbabwean*, Zimbabuan*, Zimbabwan*, Zimbabuense*, Harare]
+
+  MIDDLE:
+    'AO': [Angola, Angolan*, Angolense*, Luanda]
+    'CD': [República Democrática do Congo, RDC, Congo, Congo-Quinxassa, Congo-Quinxasa, Congolês*, Congolense*, Conguês*, Quinxassa, Quinxasa, Kinshasa]
```

Comment:
@koheiw I remember some changes to the handling of these very similar matches of CD and CG... but I can't quite remember how it worked. So, I wonder whether "Congo" should be in CD or in "CG" which is in line with en dictionary.

Reply:
I did not touch those initials, they kept the same pattern as the english dictionary where "CD" stands for the Democratic Republic Congo and "CG" is for the Congo Republic.

Reply:
Here I wasn't sure on what to do in order to make a choice so I did 2 things: I looked at how the Portuguese wikipedia handled the names and what happened when I googled for "Congo" in both the Brazilian and the Portuguese googles. 

Wikipedia only adds the word "Congo" as the name for the Democratic Republic Congo. And for the google searches the majority of the results also associated "Congo" with the Democratic Republic Congo. Those were the reasons why I made this choice.

But I am open to change, if it seems like it is best if it matches the english master.

Reply:
If "República Democrática do Congo" is in CD, "Congo" in CG is ignored by quanteda's functions because latter is the part of the former. If "Congo" is in CD and CG, both match. For this reason, "DR Congo" is for CD in the English version. Let's follow this convention.